src/js/components/molecules/product/stories/product.md

```diff
@@ -0,0 +1,4 @@
+# Product
```

Comment:
Description and links to DSM documentation are missing

Reply:
Fixed